src/store/createStore.ts

```diff
@@ -13,12 +13,14 @@ export interface Store {
 
   // Account
   account: {
-    publicKey: string;
+    publicKey: string | undefined;
+    secretKey: string | undefined;
+    registeredNetwork: Network | EmptyObj;
```

Comment:
added this to compare the current switched network to the previous network that the account was created in